main.go

```diff
@@ -78,6 +85,8 @@ func init() {
 	flag.StringVar(&instanceOverride, "instance", "", "instance: Spanner instance to use")
 	flag.StringVar(&filePrefix, "prefix", "", "prefix: file prefix for generated files")
 	flag.StringVar(&driverName, "driver", "pg_dump", "driver name: flag for accessing source DB or dump files (accepted values are \"pg_dump\", \"postgres\", \"mysqldump\", and \"mysql\")")
+	flag.StringVar(&tables, "tables", "", "tables: tables to use for migration (only for DynamoDB)")
```

Comment:
I think we should probably have this for all drivers, not just DynamoDB.

Is there some reason we specifically need this for Dynamo?

Reply:
We don't have to specify this parameter for Dynamo. In that case, all Dynamo tables under the user account will be migrated. With this `tables` parameter, user can choose to only migrate a single table or a list of tables instead of all of them. 

Yeah, we can let all drivers to support this, but currently it only supports for DynamoDB driver.

Reply:
We probably need to give some thought to flag naming if we're going to have this for all drivers.
In general, we've tried to have our flags consistent with those used by pg_dump (and to a lesser extend, mysqldump).

In this case, pg_dump uses '-t' (and the argument is a pattern).
And mysqldump just uses command arguments i.e.

  mysqldump -u username -p db_name table1_name table2_name table3_name

but it also has a flag '-tables' to indicate that all arguments should be considered tables (first arg is no longer db_name).

A couple of options:
a) drop '--tables' for the time being (assuming it isn't fundamental for Dynamo), and add it as a separate feature across all drivers.

b) check-in current code. Main drawback is it is kind of weird to have a flag that is specific to one driver (all of our other flags are generic), and we might run into issues when we make it generic.

c) come up with a flag naming schema we can use for pg and mysql, and implementing it for dynamo in this PR (with a TODO to implement it for the others).

I have a slight preference for (a) because it is simple with no loose ends, but open to other options.

Reply:
You're right. It deserves more considerations. I'll remove it from this change.